### PR TITLE
More stabilizations for query builder and relational support

### PR DESCRIPTION
**Pull request by akphi**
A lot of small improvements:
- [x] Update Dockerfile to address CVEs
- [x] Remove graph freezer and rework `PackageableElement` hash computation **(BREAKING CHANGE)**

Fixes https://github.com/finos/legend-studio/pull/285
Fixes https://github.com/finos/legend-studio/issues/270

### CODE REVIEW

File: packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/relational/RelationalInstanceSetImplementationState.ts [hunk 1]
```diff
@@ -132,7 +141,92 @@ export class RelationalPropertyMappingState extends PropertyMappingState {
...
+  setParserError(error: ParserError | undefined): void {
+    throw new UnsupportedOperationError();
+  }
+  setCompilationError(error: CompilationError | undefined): void {
+    throw new UnsupportedOperationError();
+  }
+  get fullLambdaString(): string {
+    throw new UnsupportedOperationError();
+  }
+  processSourceInformation(
```
Comment (MauricioUyaguari) [inline packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/relational/RelationalInstanceSetImplementationState.ts:205]: Fix bug leaving form mode and breaking due to one of these methods probably

File: packages/legend-studio/src/stores/ApplicationConfig.ts [hunk 2]
```diff
@@ -141,7 +131,7 @@ class ApplicationCoreOptions {
   ): ApplicationCoreOptions {
     const config = ApplicationCoreOptions.serialization.fromJson(configData);
     if (config.DEV__enableStrictRuntimeChecks) {
```
Comment (MauricioUyaguari) [inline packages/legend-studio/src/stores/ApplicationConfig.ts:133]: do we even need this config `DEV__enableStrictRuntimeChecks`

Comment (akphi) [reply packages/legend-studio/src/stores/ApplicationConfig.ts:133]: We don't but will leave it here for now, we will need it when we do https://github.com/finos/legend-studio/issues/288 I think

File: packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/relational/RelationalInstanceSetImplementationState.ts (hunk 1 above)
Comment (akphi) [reply packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/relational/RelationalInstanceSetImplementationState.ts:206]: Done!

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**